webpack/assets/javascripts/react_app/components/SearchBar/SearchBar.js

```diff
@@ -4,71 +4,40 @@ import PropTypes from 'prop-types';
 import URI from 'urijs';
 import AutoComplete from '../AutoComplete';
 import Bookmarks from '../bookmarks';
-import { noop } from '../../common/helpers';
 
-const handleSearch = (searchQuery, onSearch) => {
-  onSearch(searchQuery);
```

Comment:
why did you remove `onSearch` option?

Reply:
I think there isn't really a use of it in the SearchBar